data/anaconda.conf

```diff
@@ -62,6 +62,11 @@ ignored_packages =
 # Enable installation of latest updates.
 enable_updates = True
 
+# Set state for repos found in .treeinfo
+enable_addon_repositories_from_treeinfo = True
+enable_variant_repositories_from_treeinfo = True
+enable_optional_repositories_from_treeinfo = False
```

Comment:
We would like to have only one option that would return a list of variant types that will be enabled. Please, enable all variant types by default.

```
# List of .treeinfo variant types to enable.
# Valid items:
#
#   variant
#   optional
#   addon
#
enabled_repositories_from_treeinfo = variant optional addon

```

It is much easier to work with a list and possibly extend the supported types in the future. You don't have to implement any validation, just split the string into a list of items (see `preserved_arguments`). Then you can easily check if `repo_treeinfo.type` is in this list.